CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## v0.3.0 [2022-07-20]
+
+_What's new?_
+
+- New table added
+  - [steampipecloud_organization_workspace_member](https://hub.steampipe.io/plugins/turbot/steampipecloud/tables/steampipecloud_organization_workspace_member) ([#19](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/19))
+
+_Enhancements_
+
+- [steampipecloud_organization_workspace_mod](https://hub.steampipe.io/plugins/turbot/steampipecloud/tables/steampipecloud_workspace_mod)
```

Comment:
This format is not consistent with our plugin CHANGELOG format, can you please look at [AWS' CHANGELOG](https://github.com/turbot/steampipe-plugin-aws/blob/main/CHANGELOG.md) as an example and update all `Enhancements` notes?